packages/dom/src/dom/clean-node-list.js

```diff
@@ -113,6 +132,7 @@ export default function cleanNodeList( nodeList, doc, schema, inline ) {
 							// contains children that are block content, unwrap
 							// the node because it is invalid.
 						} else if (
+							node.parentNode &&
```

Comment:
No `!!` in front?

Reply:
The `!!` in front is only necessary to cast it to a boolean. In this case because we're just passing it as an expression to `if` TypeScript knows about "falsy" values. If we'd been assigning it to a boolean (or returning it as a boolean) we'd need to have cast it otherwise `null` would be returned in the false case.